data/category-porn

```diff
@@ -11,15 +15,16 @@ include:illusion
 include:illusion-nonofficial
 include:jable
 include:javbus
+include:javcc
 include:javdb
-include:javfinder
 include:javwide
 include:johren
-include:livejasmin
```

Comment:
The reason I removed this category is some domains are not belonging to it. But if I remove that, it will be left only one domain in that category, so I remove it and add that domain to `category-porn`.

Reply:
i see